### ISSUE TITLE
Gem::xxx deprecation warnings

### ISSUE THREAD

**Issue by ecerulm** [drnic/rubigen#8]
Hi, 

Gem::cache , Gem::source_index and Gem::SourceIndex are deprecated and rubigen uses them.

```
NOTE: Gem.cache is deprecated, use Gem::source_index. It will be removed on or after 2011-08-01.
Gem.cache called from /usr/lib/ruby/gems/1.8/gems/rubigen-1.5.6/lib/rubigen/lookup.rb:284.
NOTE: Gem.source_index is deprecated, use Specification. It will be removed on or after 2011-11-01.
Gem.source_index called from /usr/lib/ruby/site_ruby/1.8/rubygems.rb:1139.
NOTE: Gem::SourceIndex#each is deprecated with no replacement. It will be removed on or after 2011-11-01.
Gem::SourceIndex#each called from /usr/lib/ruby/gems/1.8/gems/rubigen-1.5.6/lib/
```

**Comment by bklang** [comment]
Fixed in Rubigen 1.5.7

**Comment by joes** [comment]
I am getting a similar error in 1.5.7

NOTE: Gem.source_index is deprecated, use Specification. It will be removed on or after 2011-11-01.
Gem.source_index called from /Library/Ruby/Gems/1.8/gems/rubigen-1.5.7/lib/rubigen/lookup.rb:284.
